CODEOWNERS

```diff
@@ -0,0 +1,6 @@
+# This will cause the following groups to be assigned review
+# of any opened PRs against the branches containing this file.
+
+* @puppetlabs/puppetserver-maintainers
```

Comment:
This won't work the way you expect. See https://help.github.com/en/articles/about-code-owners#codeowners-syntax.

Reply:
Oh I see, thanks. I assumed those were bullets.

Reply:
They're pattern matching, and last match for each file gets tagged.